app/controllers/subscriptions_controller.rb

```diff
@@ -3,9 +3,6 @@ class SubscriptionsController < ApplicationController
   before_action :assign_attributes
 
   def new
-    @title = GdsApi.email_alert_api
-      .get_subscriber_list(slug: @topic_id)
-      .to_h.dig("subscriber_list", "title")
```

Comment:
This also seems weirdly pointless due to: https://github.com/alphagov/email-alert-frontend/blob/1e866f29c1fd2aba8d278f38659cd4f2a24a0563/app/controllers/subscriptions_controller.rb#L61-L69

Reply:
Agreed - I tidy this up in a later commit.